# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1315076825 25200
#      Sat Sep 03 12:07:05 2011 -0700
# Node ID 875577a19d704dd0a8767186ea4a9fe3c396de3e
# Parent  756af8385e26b09e69313163b050a1e9d1696687
codesprint: Add 1 input validation test for gen_doc_cache()

gen_doc_cache.m: Add 1 input validation test.

diff --git a/scripts/help/gen_doc_cache.m b/scripts/help/gen_doc_cache.m
--- a/scripts/help/gen_doc_cache.m
+++ b/scripts/help/gen_doc_cache.m
@@ -27,16 +27,17 @@
 ##
 ## If no directory is given (or it is the empty matrix), a cache for builtin
 ## operators, etc. is generated.
 ##
 ## @seealso{lookfor, path}
 ## @end deftypefn
 
 function gen_doc_cache (out_file = "doc-cache", directory = [])
+
   ## Check input
   if (!ischar (out_file))
     print_usage ();
   endif
 
   ## Generate cache
   if (isempty (directory))
     cache = gen_builtin_cache ();
@@ -138,8 +139,13 @@ function cache = gen_builtin_cache ()
   operators = __operators__ ();
   keywords = __keywords__ ();
   builtins = __builtins__ ();
   list = {operators{:}, keywords{:}, builtins{:}};
 
   cache = create_cache (list);
 endfunction
 
+
+%% No true tests desirable for this function.
+%% Test input validation
+%!error gen_doc_cache (1)
+
